lms/djangoapps/shoppingcart/tests/test_views.py

```diff
@@ -1488,9 +1488,34 @@ def test_shopping_cart_navigation_link_in_microsite_courseware_page(self):
             self.assertEqual(resp.status_code, 200)
             self.assertNotIn('<a class="shopping-cart"', resp.content)
 
+@attr(shard=3)
```

Comment:
What does this attribute mean?

Reply:
`SHARD` is a number indicating which subset of the tests to build.
it was in ShoppingCartViewDeletedCourseTests class  because  ShoppingCartViewsTests class has same shard. And this installation provided testing of one entity in one section. 

Now I relocate test method  from ShoppingCartViewDeletedCourseTests to ShoppingCartViewsTests.